cub/test/catch2_test_device_histogram.cu

```diff
@@ -50,6 +50,7 @@
 DECLARE_LAUNCH_WRAPPER(cub::DeviceHistogram::HistogramEven, histogram_even);
 DECLARE_LAUNCH_WRAPPER(cub::DeviceHistogram::HistogramRange, histogram_range);
 
+_CCCL_SUPPRESS_DEPRECATED_PUSH
 DECLARE_TMPL_LAUNCH_WRAPPER(cub::DeviceHistogram::MultiHistogramEven,
```

Comment:
Of course the warning suppression doesn't work for the `_lid1` targets. What do the reviewers think? Do we need to cover the deprecated APIs in the unit tests? That would make solving this problem obsolete.